ivy/functional/frontends/torch/tensor.py

```diff
@@ -1091,6 +1091,23 @@ def fix_(self):
         self.ivy_array = self.fix().ivy_array
         return self
 
+    @with_unsupported_dtypes(
+        {
+            "2.0.1 and below": (
+                "bfloat16",
+                "bool",
+                "uint8",
+                "int8",
+                "int16",
```

Comment:
int16 is also supported

Reply:
But all those tests fail at a deeper level than this instance method.  What's the reason for that?

Reply:
It seems that they are not supported in certain backends? How do I change that?